### PR TITLE
Disable Cornice schema request binding (fixes #172)

### LINKED ISSUE #172: Schema binding in Cornice is costly
When disabling request binding, the time spent in this part of the code drops from 25% to 6%.

See https://github.com/mozilla-services/cornice/blob/master/cornice/schemas.py#L34
And https://github.com/mozilla-services/cliquet/blob/master/cliquet/resource.py#L83

### PR DESCRIPTION
**Pull request by leplatrem**
Requires https://github.com/mozilla-services/cornice/pull/288

### CODE REVIEW

**Comment by leplatrem** [comment]
@ametaireau r?

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 REQUIREMENTS = [
     'colander',
-    'cornice >= 0.20',  # Fixes cors.
+    'cornice >= 1.0',  # Disable request binding.
```
Comment (ametaireau) [inline setup.py]: I believe it's 1.0.0 (not sure how the comparison works).

**Comment by ametaireau** [comment]
LGTM.

**Comment by ametaireau** [comment]
Build was failing for a dependency to structlog which actually exist, so it's a false negative. Merging.

**Pull request merged**